### DISCUSSION TITLE
Permission Filtering on User-Fields

### DISCUSSION THREAD

**Discussion by LeonNonnast** [directus/directus#8330] [Archive]
Hi experts!

I'd like to filtering collection values for a specific user role by using user field values.

For Example:

Users has a many to many relation to the collection office. 

Each Office writes Blog-Posts.

Now, I want to hide Posts from other offices. 

Is there a way to access users fields in permission? 

```json
{
 "office": {
     "_in" : "users.offices"
  }
}
```

Best Regards and many thanks!

Leon

**Comment by rijkvanzanten** [comment]
Heya! This is a duplicate of https://github.com/directus/directus/discussions/2993
